docs/ref/contrib/sitemaps.txt

```diff
@@ -530,3 +543,13 @@ Once the sitemaps application is added to your project, you may also
 ping Google using the ``ping_google`` management command::
 
     python manage.py ping_google [/sitemap.xml]
+
+The management command accepts the following optional arguments:
+
+    .. versionadded:: 2.2
+
+    * ``-d --site_domain``, If you are not using ``django.contrib.sites``, use this
```

Comment:
Do we prefer dashes to underscore in command options? (So `--site-domain` etc?)

Reply:
Dashes. Definitely.

Reply:
+1

Reply:
These should be documented using `.. django-admin-option::` as in `docs/ref/django-admin.txt` (and then linked from the release notes).

Reply:
I would also prefer dashes here, but I used underscore for consistency reasons. If you check the other argument, `sitemap_url` => https://github.com/django/django/pull/10651/files#diff-53e32f908b617b1896a4aacca34b4b25R9

Reply:
Hey @timgraham 

Sorry, but I'm confused about linking the option from release notes in `sitemaps.txt` documentation. Can you please help me here? I've addressed all other reviews.